AU2/plugins/CorePlugin.py

```diff
@@ -216,7 +225,6 @@ def on_assassin_request_update(self, assassin: Assassin):
             InputWithDropDown(self.html_ids["Water Status"], "Water Status", WATER_STATUSES, selected=assassin.water_status),
             InputWithDropDown(self.html_ids["College"], "College", COLLEGES, selected=assassin.college),
             LargeTextEntry(self.html_ids["Notes"], "Notes", default=assassin.notes),
-            #Checkbox(self.html_ids["Police"], "Police? (y/n)", checked=assassin.is_police)

```

Comment:
Maybe leave another comment explaining why this is commented, or just remove the comment altogether. Not obvious why it's there otherwise

Reply:
(The change here *is* removing the comment (that's what the - and the red mean).)

Reply:
Oh oops